src/app/search/_components/result.tsx

```diff
@@ -0,0 +1,605 @@
+                </Skeleton>
+                <Skeleton>
+                  <Text>Content</Text>
+                </Skeleton>
+              </Stack>
+            ))}
+          </Flex>
+        </Flex>
+      </Paper>
+    );
+
+  const { label, colour, link, tableComponent: Component } = resultType[item.type];
+
+  return (
+    <Paper component={Link} href={link(item)} className={classes.result} radius="sm" bg={`${colour}.0`}>
+      <Flex pl="lg" direction="row" gap="xl">
+        <Center>
+          <Paper w={150} py={5} px={15} bg={`${colour}.2`} radius="xl">
+            <Center>
+              <Text size="sm" fw="bold" c={`${colour}.9`}>
+                {label}
+              </Text>
+            </Center>
+          </Paper>
+        </Center>
+        <Component item={item} />
+      </Flex>
+    </Paper>
+  );
+}
+
+export function CardResult({ item }: { item?: Item }) {
+  if (!item)
+    return (
+      <Paper h="100%" radius="lg" style={{ border: `1px solid var(--mantine-color-midnight-0)` }}>
+        <Flex direction="column" gap="sm" h="100%">
+          <Group
+            p="md"
+            justify="space-between"
+            bg="gray.0"
+            style={{
+              borderTopLeftRadius: "var(--mantine-radius-lg)",
+              borderTopRightRadius: "var(--mantine-radius-lg)",
+            }}
+          >
+            <Skeleton radius="xl">
+              <Paper py={5} px="xl" radius="xl">
+                <Center>
+                  <Text size="sm">Loading</Text>
+                </Center>
+              </Paper>
+            </Skeleton>
+          </Group>
+          <Box p="md" pb="md" h="100%">
+            <Skeleton h={200} radius="lg" />
+          </Box>
+        </Flex>
+      </Paper>
+    );
+
+  const { label, colour, link, cardComponent: Component } = resultType[item.type];
+
+  return (
+    <Paper
+      component={Link}
+      href={link(item)}
+      className={classes.card}
+      h="100%"
+      radius="lg"
+      style={{ border: `1px solid var(--mantine-color-${colour}-1)` }}
+    >
+      <Flex direction="column" gap="sm" h="100%">
+        <Group
+          p="md"
+          justify="space-between"
+          bg={`${colour}.0`}
+          style={{ borderTopLeftRadius: "var(--mantine-radius-lg)", borderTopRightRadius: "var(--mantine-radius-lg)" }}
+        >
+          <Paper py={5} px="xl" bg={`${colour}.2`} radius="xl">
+            <Center>
+              <Text size="sm" fw="bold" c={`${colour}.9`}>
+                {label}
+              </Text>
+            </Center>
+          </Paper>
+        </Group>
+        <Box px="md" pb="md" h="100%">
+          <Component item={item} />
+        </Box>
+      </Flex>
+    </Paper>
+  );
+}
+
+interface ResultsProps {
+  items?: Item[];
+  perPage: number;
+  layout: TableCardLayout;
+}
+
+export function Results({ items, perPage, layout }: ResultsProps) {
+  const results = uniqBy(items, (item) => Object.values(item).join("-"));
```

Comment:
is this a guard to eliminate duplicates? you should be able to remove this with the latest backend (you'll need to do a search reindex). otherwise you'll get inconsistent page sizes if its filtered out on the frontend.